internal/tiltfile/tiltfile_test.go

```diff
@@ -5136,6 +5116,22 @@ func disablePush(disable bool) disablePushHelper {
 	return disablePushHelper{disable}
 }
 
+type updateCmdHelper struct {
+	cmd string
+}
+
+func updateCmd(cmd string) updateCmdHelper {
+	return updateCmdHelper{cmd}
+}
+
+type ltHelper struct {
```

Comment:
can we spell this out as localTarget? i'm not sure i'd ever remember what lt meant without digging around. i would assume it meant "less than"